### PR TITLE
Minio base support

### PR DESCRIPTION
**Pull request by siarhei-shliayonkin**
- MinIO standalone server deployment
- minio package w/ base Save/Load functions
- CreateBucketIfNotExists()

### CODE REVIEW

File: pkg/minio/minio.go [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+		}
+		instance = &Storage{
+			client: client,
+			log:    log,
+		}
+	})
+	return instance, nil
+}
+
+const BucketName = "integrity"
```
Comment (alavrovinfb) [inline pkg/minio/minio.go:94]: Maybe, it will be better to have a configurable bucket name? e.g. from flags or ENVs.

File: pkg/minio/minio.go [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+	}
+	s.log.WithFields(logrus.Fields{
+		"objectName": objectName,
+		"size":       info.Size,
+	}).Debug("uploaded successfully")
+	return nil
+}
+
+// Load loads and returns data from the bucket for the @objectName
+func (s *Storage) Load(ctx context.Context, objectName string) ([]byte, error) {
```
Comment (alavrovinfb) [inline pkg/minio/minio.go]: Maybe, it will be better to pass bucket name as function argument rather use global?

File: pkg/minio/minio.go [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+			log:    log,
+		}
+	})
+	return instance, nil
+}
+
+const BucketName = "integrity"
+
+// Save stores @data into the bucket with the given @objectName
+func (s *Storage) Save(ctx context.Context, objectName string, data []byte) error {
```
Comment (alavrovinfb) [inline pkg/minio/minio.go]: Maybe, it will be better to pass bucket name as function argument rather use global?

File: pkg/minio/minio.go [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+	MsgFailedUpload         string = "failed to upload object: %w"
+	MsgFailedLoad           string = "failed to load object: %w"
+	MsgFailedGetInfo        string = "failed to get info for object: %w"
+	MsgFailedCreateBucket   string = "failed to create bucket: %w"
+)
+
+func init() {
+	fsMinIO := pflag.NewFlagSet("minio", pflag.ExitOnError)
+	fsMinIO.Bool("minio-enabled", false, "enable MinIO")
+	fsMinIO.String("minio-host", "minio.svc.local:9001", "MinIO host")
```
Comment (alavrovinfb) [inline pkg/minio/minio.go]: I suppose, port `9001` is service UI port, correct Minio backend port is `9000` isn't it?

**Comment by alavrovinfb** [comment]
Overall. I'm not sure if we should merge these changes into the dev branch until we've made a decision about minio. Maybe we should keep these changes in feature branch for now? Thoughts?

**Review by alavrovinfb — APPROVED** [review]

File: pkg/minio/minio.go (hunk 1 above)
Comment (siarhei-shliayonkin) [reply pkg/minio/minio.go:94]: Already done. This is just default value.

**Pull request merged**